app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
-<h2>User information and posts</h2>
\ No newline at end of file
+<div class="userContainer flex flex-col gap-8 py-8 w-[80%] mx-auto">
+  <div class="userCard">
+    <a href="#" class="flex flex-row gap-[30px]">
+      <div class="userImage w-[15%]">
+        <img src=<%= @user.photo %>>
+      </div>
+      <div class="userDetails bg-[#11537d] rounded-[30px] flex-grow text-white flex justify-between items-center p-[30px]">
+        <h2 class="text-[40px] font-[600]"><%= @user.name %></h2>
+        <p class="self-end text-[20px] font-[400]">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </a>
+  </div>
+  <p class="userBio bg-[#11537d] rounded-[30px] flex-grow text-3xl font-semibold text-white flex justify-between items-center p-[30px]">
+    <%= @user.bio %>
+  </p>
+  <ul class="userCards flex flex-col gap-8">
+    <% @posts.each do |post| %>
+      <li class="postCard">
```

Comment:
- [ ] Just like the previous reviewer suggested, in the users show page, you are meant to use the method you created in your model to show only the last 3 posts of that user. Currently, you are mapping over all the posts and that's not correct. :wave: here is a suggestion of whats required. :+1: 
```ruby
   <% @user.last_three_posts do |post| %>
     // Your erb templates here
   <% end %>
```

Reply:
Can be improved!